src/index.js

```diff
@@ -58,7 +58,9 @@
  * Represents a shoveled plot
  * @typedef farmhand.shoveledPlot
  * @type {Object}
- * @property {boolean} wasShoveledToday
+ * @property {boolean} wasShoveled
```

Comment:
Looking at this all again, what do you think about renaming this to `isShoveled`? It's a subtle (and perhaps a little annoying to make) semantic tweak that I think helps make the logic a little bit clearer since plots can remain shoveled for many days.

Sorry I didn't think of this before. Seeing this being used more in the code made me think of it! 😅

Reply:
no worries, i think `isShoveled` makes more sense. updating